darshan-util/pydarshan/docs/install.rst

```diff
@@ -1,53 +1,67 @@
 .. highlight:: shell
 
+.. _installation:
+
 ============
 Installation
 ============
 
 
-Stable release
---------------
+Stable release (PyPI)
+---------------------
 
-To install PyDarshan, run this command in your terminal:
+The preferred method for installing PyDarshan is to use `pip`_ to install stable
+releases directly from PyPI:
 
 .. code-block:: console
 
     $ pip install darshan
 
-This is the preferred method to install PyDarshan, as it will always install the most recent stable release.
-If you don't have `pip`_ installed, this `Python installation guide`_ can guide
-you through the process.
+PyDarshan releases on PyPI include manylinux and macOS wheels that simplify
+distribution of PyDarshan and its dependencies for most users.
 
 .. _pip: https://pip.pypa.io
-.. _Python installation guide: https://www.mcs.anl.gov/research/projects/darshan/
 
 
-PyDarshan assumes that a recent 'darshan-utils' is installed as a shared
-library. If darshan-util is not installed consult with the darshan
-documentation or consider using `Spack`_ to install::
+From Spack
+----------
+
+PyDarshan is also available within the `Spack`_ package manager.
 
-    spack install darshan-util
+.. code-block:: console
 
+    $ spack install py-darshan
 
 .. _Spack: https://spack.io/
 
 
 From sources
 ------------
 
-The sources for PyDarshan can be downloaded from the `Github repo`_.
-
-You can either clone the public repository:
+The sources for PyDarshan can be obtained as part of the `Darshan Github repo`_, as shown below.
 
 .. code-block:: console
 
     $ git clone https://github.com/darshan-hpc/darshan.git
     $ cd darshan/darshan-util/pydarshan
 
+Users can then use ``pip`` to install the PyDarshan package from source.
+
+.. code-block:: console
+
+    $ pip install .
+
+When building PyDarshan from sources, users need to make a copy of the darshan-util shared
+library available, typically using `LD_LIBRARY_PATH`.
```

Comment:
This won't suffice on Macs though, FWIW. I know some projects kind of use "tabs" or tables to show the install flows for different platforms--we probably don't have the bandwidth for that, but could mention the env variables we use for Mac in our GHA CI as a start.